### PR TITLE
Mobilefirst approach feature  branch completed

### PR DESCRIPTION
**Pull request by flov3rh**
Created top  navbar and about me section in the webpage, following the design of template 3 and successfully setup with no linter errors.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+             you need coded do not hesitate to contact me.
+          </p>
+          <ul class="list-format font-color">
+            <li><a href="https://github.com/flov3rh" class="fa-brands fa-square-github"></a></li>
+            <li><a href="https://www.linkedin.com/in/flov3rh/" class="fa-brands fa-linkedin"></a></li>
+            <li><a href="https://twitter.com/flov3rh" class="fa-brands fa-twitter"></a></li>
+            <li><a href="#" class="fa-solid fa-envelope"></a></li>
+          </ul>
+        <a href="#middle" id="down-button" class="fa-solid fa-chevron-down"></a>
+      </header>
```
Comment (zieeco) [inline index.html]: - Hey, the container of this content should be a `section` tag is this is the first section of your document to enforce a good use of the html5 semantic tag. please, kindly update this.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+             for, I have a passion for software development, please review some of
+             my works and experience! if you like what you see and have a project
+             you need coded do not hesitate to contact me.
+          </p>
+          <ul class="list-format font-color">
+            <li><a href="https://github.com/flov3rh" class="fa-brands fa-square-github"></a></li>
+            <li><a href="https://www.linkedin.com/in/flov3rh/" class="fa-brands fa-linkedin"></a></li>
+            <li><a href="https://twitter.com/flov3rh" class="fa-brands fa-twitter"></a></li>
+            <li><a href="#" class="fa-solid fa-envelope"></a></li>
+          </ul>
```
Comment (zieeco) [inline index.html]: - hey, the social media icon looks different from that on Figma. please, kindly use the same icon as used in the Figma design to make your design consistent following the given template.


image


image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  </head>
+  <body>
+    <div class="background">
+      <nav>
+        <ul class="list-format font-color">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </nav>
```
Comment (zieeco) [inline index.html]: - Let's wrap the nav in the header as it's semantically the container used to wrap the introductory content of a webpage to improve your semantic usage.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    <script src="https://kit.fontawesome.com/cebf050324.js" crossorigin="anonymous"></script>
+  </head>
+  <body>
+    <div class="background">
+      <nav>
+        <ul class="list-format font-color">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
```
Comment (zieeco) [inline index.html]: - hey, in the mobile version, theses links should not be visible. let's add the hamburger icon and make these links not visible for the mobile version. see the Figma template to update this change.

File: css/styles.css [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+  display: flex;
+  justify-content: space-around;
+  margin: auto;
+  padding: 0;
+  width: 30%;
+}
+
+.font-color {
+  color: #172b4d;
+}
```
Comment (zieeco) [inline css/styles.css]: - kindly add a line-height: 24px to the body or to this class to improve the spacing in your text to improve your design.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @flov3rh,




Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions

- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (flov3rh) [reply index.html]: image

the template shows me a different order, all icons are linked to my accounts however I do not have an account with the peace sign logo hence why was omitted and the github logo is not available in fontawesome so I had to use a different one.

**Review by Cabraham1 — APPROVED** [review]
Hi @flov3rh , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**